src/flow.ts

```diff
@@ -514,15 +524,25 @@ export class Flow {
   setLocalFlag(index: i32, flag: LocalFlags): void {
     if (index < 0) return;
     let localFlags = this.localFlags;
-    let flags = index < localFlags.length ? unchecked(localFlags[index]) : 0;
+    let flags = 0;
+    if (index < localFlags.length) {
+      flags = unchecked(localFlags[index]);
+    } else {
+      localFlags.length = index + 1;
```

Comment:
These changes to `setLocalFlag` and below to `unsetLocalFlag` seem redundant now with #2586 merged, thast is with unchecked indexed access on normal arrays removed / fixed, in that the assignment below now implies updating length.

Reply:
I removed this change as well.